src/Features/CSharp/Portable/CodeFixes/GenerateMethod/GenerateMethodCodeFixProvider.cs

```diff
@@ -80,6 +83,12 @@ protected override SyntaxNode GetTargetNode(SyntaxNode node)
                 return memberBindingExpression.Name;
             }
 
+            var assignmentExpression = node as AssignmentExpressionSyntax;
```

Comment:
Is this so that it works with the caret outside of the parentheses?  I'm not sure it's important to support that case.